.changelog/18140.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+cloud: Removes requirement for HCP to provide a management token
```

Comment:
```suggestion
hcp: Removes requirement for HCP to provide a management token
```
I skimmed the [changelog](https://github.com/hashicorp/consul/blob/main/CHANGELOG.md) and it looks like we opt for "hcp". Let me know though if you see otherwise